quickfixj-core/src/test/java/quickfix/SessionSettingsTest.java

```diff
@@ -537,4 +599,8 @@ public void testConcurrentAccess() throws ConfigError, InterruptedException {
         return defaultSettings;
     }
 
+    protected String getConfigurationFileName() {
```

Comment:
Make this private - it's not used outside this class.

Reply:
Fixed